### PR TITLE
[13.0][MIG] event_track_location_overlap: Migration to 13.0

### PR DESCRIPTION
**Pull request by victoralmau**
Migration to 13.0

Please @ernestotejeda and @pedrobaeza can you review it?

@Tecnativa TT27583

### CODE REVIEW

**Review by pedrobaeza — COMMENTED** [review]
Let's avoid the use of oca.decorators library, as it's very marginal. Remove the decorator `@foreach()`, and you call such method `_check_overlappable_one`, and let the other with the constraint for call record per record the other one.

File: event_track_location_overlap/models/event_track_location.py [hunk 1]
```diff
@@ -14,15 +12,14 @@ class EventTrackLocation(models.Model):
     overlappable = fields.Boolean(help="Can this location have simultaneous tracks?")
 
     @api.constrains("overlappable")
```
Comment (pedrobaeza) [inline event_track_location_overlap/models/event_track_location.py:14]: You need also to split this for multi record support

Comment (victoralmau) [reply event_track_location_overlap/models/event_track_location.py:14]: Ok, if I have understood it correctly, the last change would be correct?

Comment (pedrobaeza) [reply event_track_location_overlap/models/event_track_location.py:14]: Yes.

**Review by pedrobaeza — APPROVED** [review]

**Review by ernestotejeda — APPROVED** [review]

**Pull request merged**